### PR TITLE
🌿 Update README, .fernignore and publish action

### PR DESCRIPTION
**Pull request by armandobelardo**

### CODE REVIEW

File: .fernignore [hunk 1]
```diff
@@ -1 +1,6 @@
 # Specify files that shouldn't be modified by Fern
+
+.github/workflows/publish.yml
+test/test_helper.rb
```
Comment (dvargas92495) [inline .fernignore:4]: What are these two test files?

File: vellum_ai.gemspec [hunk 2]
```diff
@@ -4,7 +4,7 @@ require_relative "lib/gemconfig"
 
 Gem::Specification.new do |spec|
   spec.name = "vellum_ai"
-  spec.version = "v0.3.0"
+  spec.version = "0.3.0"
```
Comment (dvargas92495) [inline vellum_ai.gemspec:7]: Does ruby require that we don't prefix with the `v`?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+## Async Client
+
+```ruby
+require "vellum_ai"
+
+
+client = Vellum::AsyncClient.new(api_key: "YOUR_API_KEY")
+
+result = client.generate(
```
Comment (dvargas92495) [inline README.md:52]: We'll want execute_prompt here and above instead - will push an edit here in a few

File: vellum_ai.gemspec (hunk 2 above)
Comment (armandobelardo) [reply vellum_ai.gemspec:7]: It's regular semvar! So yes, no v prefix, if it's a big change for you I can look to make the change within the generator itself

File: .fernignore (hunk 1 above)
Comment (armandobelardo) [reply .fernignore:4]: They are meant to be test stubs should people want to easily play around with the gem, they're programmatically generated and put into every ruby SDK in the latest version of the generator.
It also makes it easy to add test in the future should people want to

**Pull request merged**